js/templates/modals/report.html

```diff
@@ -16,20 +16,20 @@ <h1 class="h3 txCtr"><%= ob.polyT('listingReport.title') %></h1>
         <% } else { %>
           <form class="flexCol flexExpand gutterV compact">
             <div class="btnRadio clrBr">
-              <input type="radio" name="reason" value="Offensive" checked id="Offensive">
+              <input type="radio" name="reason" value="Offensive" id="Offensive" <% if (ob.reason === 'Offensive') print('checked') %>>
```

Comment:
It's incredibly minor and I wouldn't even change it, but it just seems odd to make the Value have that type of capitalization.